parking_permits/customer_permit.py

```diff
@@ -193,8 +193,7 @@ def create(self, address_id, registration):
                 primary_permit = self.customer_permit_query.get(primary_vehicle=True)
                 contract_type = primary_permit.contract_type
                 primary_vehicle = not primary_permit.primary_vehicle
-                if contract_type == FIXED_PERIOD:
```

Comment:
Why this if is removed?

Reply:
Previously there was no need to have end time for open ended permits. But now when we need to increase permit end time on a monthly basis, it is more logical to simply have that end time as well.